market/connector/microsoft365/calendar/meta.json

```diff
@@ -1,13 +1,28 @@
 {
-	"id": "ms-calendar",
+	"id": "msgraph-calendar",
 	"version": "1.0",
 	"name": "Microsoft Calendar",
 	"description": "Microsoft Calendar is part of the Microsoft 365 API that lets you manage emails and contacts.",
 	"type": "connector",
 	"platformReview": "4.5",
-	"sourceUrl": "https://github.com/axonivy/market",
+	"sourceUrl": "https://github.com/axonivy-market/msgraph-connector",
 	"language": "English",
 	"industry": "Cross-Industry",
-	"compatibility": "9.2+",
-	"tags": ["office"]	
+	"compatibility": "9.4+",
+	"tags": ["office"],
+	"mavenArtifacts": [
+		{
```

Comment:
please adapt from an existing sibling ... e.g. docuSign:
- ZIP product artifact is missing
- use the same in the 'mavenArtifacts' order (1. zip, 2. connector, 3. demo)

Reply:
I guess the same applies for the other meta.json files